dev.py

```diff
@@ -779,7 +780,8 @@ def scipy_tests(cls, args, pytest_args):
                 args.mode,
                 verbose=verbose,
                 extra_argv=extra_argv,
-                doctests=False,
+                # doctests=False,
+                doctests=args.doctests,
```

Comment:
Also this

Reply:
We'll need to not forget to document in big bold letters that without the plugin `pytest --doctest-modules` runs both tests and doctests, while with the plugin, it's only doctests.

Reply:
Yes, I have opened an issue on updating the documentation: https://github.com/ev-br/scpdt/issues/96